src/i_winmusic.c

```diff
@@ -715,7 +717,7 @@ static void SendEMIDI(unsigned int delta_time, const midi_event_t *event,
     unsigned int flag;
     int count;
 
-    switch ((int)event->event_type)
+    switch (event->data.channel.param1)
```

Comment:
Made a mistake copying this section way back in this PR: https://github.com/fabiangreffrath/woof/pull/1053/commits/e2bf6fb0d80133342b71acb766c9409d9ddd7a9a
I'll need to make a Chocolate PR after I'm done with this one.